packages/atlas/src/components/NumberFormat/NumberFormat.tsx

```diff
@@ -68,14 +86,69 @@ export const NumberFormat = forwardRef<HTMLHeadingElement, NumberFormatProps>(
         variant={variant}
         ref={mergeRefs([ref, textRef])}
       >
+        {withDenomination === 'before' && (
+          <Text
+            className="denomination"
+            as="span"
+            color={bnValue.isNeg() || isNegative ? 'colorTextError' : 'colorText'}
+            variant={variant}
+          >
+            ({formattedDenominatedValue !== '<$0.01' ? '$' : ''}
+            {formattedDenominatedValue}){' '}
+          </Text>
+        )}
         {displayedValue || formattedValue}
         {withToken && ` ${atlasConfig.joystream.tokenTicker}`}
+        {withDenomination === 'after' && (
+          <Text
+            className="denomination"
+            as="span"
+            color={bnValue.isNeg() || isNegative ? 'colorTextError' : 'colorText'}
+            variant={variant}
+          >
+            {' '}
+            ({formattedDenominatedValue !== '<$0.01' ? '$' : ''}
+            {formattedDenominatedValue}){' '}
+          </Text>
+        )}
       </StyledText>
     )
 
     return (
       <>
-        {content}
+        {withDenomination ? (
+          <Container orientation={withDenomination}>
+            {icon ? (
+              <IconContainer>
+                {icon}
+                {content}
+              </IconContainer>
+            ) : (
+              content
+            )}
+            {!TEXT_DENOMINATION_ALIGMENTS.includes(withDenomination) && (
+              <Denomination
+                align={denominationAlign}
+                className="denomination"
+                as="span"
+                color={bnValue.isNeg() || isNegative ? 'colorTextError' : 'colorText'}
+                variant="t100"
+                ref={mergeRefs([ref, textRef])}
+              >
+                {formattedDenominatedValue !== '<$0.01' ? '$' : ''}
+                {formattedDenominatedValue}
```

Comment:
``` 
{formattedDenominatedValue !== '<$0.01' ? `$${formattedDenominatedValue}` : ''}
``` 
?

Reply:
Formatted value can be just a number `100` etc. or `<0.01` with JOY there is no problem since the ticker is behind. For USD ticker is in front, so for normal value we just add `$`, but for lower than 0.01 we insert `$` in the switch case, so we cannot add it here as well. It's either this conditional or adding `$` in every switch case